src/options/options_public_template.cpp

```diff
@@ -502,15 +502,14 @@ std::string get(const Options& options, const std::string& name)
 {
   Trace("options") << "Options::getOption(" << name << ")" << std::endl;
   ${getoption_handlers}$
-
-  throw UnrecognizedOptionException(name);
+  throw OptionException("Unrecognized informational or option key or setting: " + name);
```

Comment:
"informational"?

Reply:
I have no clue what I wanted the message to say there... I'll fix it.